src/frontend/apps/impress/src/i18n/translations.json

```diff
@@ -180,20 +191,17 @@
       "Copied to clipboard": "Copié dans le presse-papier",
       "Copy as {{format}}": "Copier en {{format}}",
       "Copy link": "Copier le lien",
-      "Copyright": "Copyright",
       "Correct": "Corriger",
       "Defender of Rights - Free response - 71120 75342 Paris CEDEX 07": "Défenseur des droits - Réponse gratuite - 71120 75342 Paris CEDEX 07",
       "Delete": "Supprimer",
       "Delete a doc": "Supprimer un doc",
       "Delete document": "Supprimer le document",
       "Doc visibility card": "Carte de visibilité du doc",
-      "Docs": "Docs",
       "Docs Logo": "Logo Docs",
       "Docs: Your new companion to collaborate on documents efficiently, intuitively, and securely.": "Docs : Votre nouveau compagnon pour collaborer sur des documents efficacement, intuitivement et en toute sécurité.",
       "Document owner": "Propriétaire du document",
       "Document title updated successfully": "Titre du document mis à jour avec succès",
       "Download": "Télécharger",
-      "E-mail:": "E-mail:",
```

Comment:
It was lacking some french translations on `Crowdin`, I just added them, is there a way to replay this PR or we have to make another one ?

Reply:
We can pull this PR in local otherwise and update it from there. What is the best way ?

Reply:
I triggered the workflow and the PR has been updated. Can you check the new trads are here ?